element_miniscope/imaging.py

```diff
@@ -88,8 +88,8 @@ class ProcessingParamSet(dj.Lookup):
     """
 
     @classmethod
-    def insert_new_params(cls, processing_method: str, paramset_idx: int,
-                          paramset_desc: str, params: dict):
+    def insert_new_params(cls, processing_method: str, package_version: str,
+                          paramset_idx: int, paramset_desc: str, params: dict):
```

Comment:
we don't need `package_version` here, as we discussed `package_version` should be at the `Processing` level, right?

Reply:
Yes,  including the version in `Processing` level is a bit better because users do not need to be aware of the package version and it is auto detected.

Reply:
Good point.  Moved `package_version` to `imaging.Processing`.